File: k번째 약수/qyur.py
import sys
sys.stdin = open("./k번째약수.txt",'r')

N, K = map(int, input().split())

arr = []
for i in range(1, N + 1):
    if N % i == 0: # 약수면 추가
        arr.append(i)

if K > len(arr): # 약수 모음 개수가 더 작은 경우
    print(0)
else:
    print(arr[K-1])

# 파이썬 적응안된다 ... 예전엔 파이썬이 제일 쉽다고 생각했는데 ...

File: Section-3-04_두리스트합치기/qyur.js
const fs = require("fs");

const getParsedData = (i) => {
  const input = fs.readFileSync(`./in${i}.txt`, "utf8").toString().split('\n');

  const parsedList = input.map((arr, index) => {
    if (index !== 1 && index !== 3) return null;

    return arr.split(' ').map((number)=> Number(number)).filter(n => n);
  }).filter(l=>l)

  return parsedList;
}

for (let i = 1; i < 6; i += 1) {
  const [firstList, secondList] = getParsedData(i);
  console.log(`#${i}`, [...firstList,...secondList].sort((a,b) =>  a - b).join(' '));
}